satpy/readers/ahi_hsd.py

```diff
@@ -33,6 +33,27 @@
 ``observation_end_time`` keys as well as the ``start_time`` and ``end_time``
 keys.
 
+Satellite Position
+******************
+
+As discussed in the :ref:`orbital_parameters` documentation, a satellite
+position can be described by a specific "actual" position, a "nominal"
+position, a "projection" position, or sometimes a "nadir" position. Not all
+readers are able to produce all of these times. In the case of AHI HSD data
```

Comment:
Typo: ... are able to produce all of these _positions_

Reply:
Done.